src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.image-link {
+  display: block;
+  height: 40px;
+  padding: 10px 0;
+  margin-left: 50px;
```

Comment:
Strange solution, you can just set horizontal paddings to parent container instead of margins for both childs